doc-content/enterprise-only/migration/migration-tool-niogit-7.4-proc.adoc

```diff
@@ -0,0 +1,217 @@
+      "url":"https://origin-repository.jboss.org/nexus/content/groups/kie-internal-group",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":false,
+      "snapshotUpdatePolicy":"never"
+    }
+  ],
+
+  "repositories-remove":[
+    {
+      "id":"productization-repository", "url":"http://download.lab.bos.redhat.com/brewroot/repos/jb-ip-6.1-build/latest/maven/"
+    }
+  ],
+
+  "repositories-update-urls":[
+    {
+      "id":"guvnor-m2-repo", "url":"http://localhost:8080/{URL_COMPONENT_CENTRAL}/maven3/"
+    }
+  ],
+
+  "pluginRepositories-add":[
+
+    {
+      "id":"jboss-public-repository-group",
+      "name":"JBoss Public Repository Group",
+      "url":"http://repository.jboss.org/nexus/content/groups/public/",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":true,
+      "snapshotUpdatePolicy":"never"
+    },
+    {
+      "id":"kie-internal-staging-repository-group",
+      "name":"KIE Internal Staging Repositories",
+      "url":"https://origin-repository.jboss.org/nexus/content/groups/kie-internal-group",
+      "releasesEnabled":true,
+      "releasesUpdatePolicy":"never",
+      "snapshotEnabled":false,
+      "snapshotUpdatePolicy":"never"
+    }
+
+  ],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+
+Example `pom-migration.json` file with `dependencies` only:
+
+[source,json]
+----
+{
+  "dependencies":[
+
+    {"groupId":"junit", "artifactId":"junit", "version":"4.12", "scope":"test"},
+    {"groupId":"com.thoughtworks.xstream", "artifactId":"xstream", "version":"4.12", "scope":"test"}
+
+  ],
+  "repositories-add":[],
+  "repositories-remove":[],
+  "repositories-update-urls":[],
+  "pluginRepositories-add":[],
+  "pluginRepositories-remove":[],
+  "pluginRepositories-update-urls":[]
+}
+----
+--
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product:* {PRODUCT_SHORT}
+* *Version:* 7.5
+
+. Download *{PRODUCT} 7.5 Add-Ons* and extract the downloaded `{PRODUCT_INIT}-7.5.0-add-ons.zip` file to a temporary directory.
+. In the extracted `{PRODUCT_FILE}-add-ons` folder, extract the `{PRODUCT_INIT}-7.5-migration-tool.zip` sub-folder. The migration tool is in the `bin` directory.
+. In your command terminal, navigate to the temporary directory where you extracted the `{PRODUCT_INIT}-7.5-migration-tool` folder and run the migration tool. The `${PRODUCT_INIT_CAP}_NIOGIT_DIR` portion is the path to the `.niogit` directory that you previously copied to the {PRODUCT} 7.5 installation.
++
+--
+On Linux or UNIX-based systems:
+[source,subs="attributes+"]
+----
+$ cd $INSTALL_DIR/{PRODUCT_INIT}-7.5-migration-tool/bin
+$ ./migration-tool.sh -t ${PRODUCT_INIT_CAP}_NIOGIT_DIR
+----
+
+On Windows:
```

Comment:
"On Windows systems" if you want to follow IBM convention (and I'm writing this as a former IBM information developer who fixed this very thing in a number of peer reviews back there)